etc/search-ui-react.api.md

```diff
@@ -347,6 +361,7 @@ export interface GenerativeDirectAnswerCssClasses {
 export interface GenerativeDirectAnswerProps {
     answerHeader?: string | JSX.Element;
     CitationCard?: (props: CitationProps) => JSX.Element | null;
+    CitationsComponent?: (props: CitationsProps) => JSX.Element | null;

```

Comment:
nit: the `Component` suffix is not really needed as it's implied by its type (e.g. like CitationCard), can we rename this?